src/components/floating-menu/floating-menu.js

```diff
@@ -59,6 +60,27 @@ export const getFloatingPosition = ({
   const refCenterHorizontal = (refLeft + refRight) / 2;
   const refCenterVertical = (refTop + refBottom) / 2;
 
+  if (componentsX) {
+    return {
```

Comment:
Hi @tw15egan wondering if this change is intended for tooltip in addition to overflow menu. In our early experimental effort, we used offset adjustment callbacks in [overflow menu](https://github.com/IBM/carbon-components/blob/v9.28.0/src/components/overflow-menu/overflow-menu.js#L29-L58) and [tooltip](https://github.com/IBM/carbon-components/blob/v9.28.0/src/components/tooltip/tooltip.js#L12-L49) to accommodate geometry difference in experimental style.

Reply:
Ah, I see. I will try moving that logic over to `OverflowMenu.js`